src/data/mv_polynomial/basic.lean

```diff
@@ -376,6 +376,11 @@ lemma support_add : (p + q).support ⊆ p.support ∪ q.support := finsupp.suppo
 lemma support_X [nontrivial R] : (X n : mv_polynomial σ R).support = {single n 1} :=
 by rw [X, support_monomial, if_neg]; exact one_ne_zero
 
+@[simp] lemma support_zero : (0 : mv_polynomial σ R).support = ∅ := rfl
+
+lemma support_sum {α : Type*} {s : finset α} {f : α → mv_polynomial σ R} :
+  (∑ x in s, f x).support ⊆ s.sup (λ x, (f x).support) := finsupp.support_sum'
```

Comment:
Thinking about it, we should maybe be using `bUnion` instead of `sup` here. Thoughts, @YaelDillies?

Reply:
The issue is independent from this PR. We have two choices:
* maintain `finset.bUnion` as the `finset` specialization of `finset.sup`. This means API duplication for functionality and glue in places.
* delete once and for all `finset.bUnion`. This means API duplication for syntax, supposedly not more than between `finset` and `generalized_boolean_algebra` for example.

I'm for the second option. So I would say this is written the correct way. Given current mathlib, using `bUnion` is fine too.

Reply:
If bUnion really is the finset specialization, then I think we should use it here; while it exists.